### ISSUE TITLE
UI

### ISSUE THREAD

**Issue by SurvivCreator** [SurvivCreator/Battlefront-Redux#3]
Need a more updated UI with settings and a streamer showcase section.

**Comment by Coolaction77** [comment]
Alright, by streamer showcase section what do you mean? Do you mean like an area that shows active streams like in Surviv.io?

**Comment by SurvivCreator** [comment]
Not necessarily, I was thinking that we should have a section where we 'advertise' YouTubers, twitch streamers, etc. who have made a video of us. Kind of like that Jurassic Park Builder concept.

**Comment by Coolaction77** [comment]
Ah, ok! I'll also add a contacting area, then, so that if someone makes a video they can tell us.

**Comment by Coolaction77** [comment]
Also I'm thinking about adding achievements, I made a list of some in the backdrop sprite.

**Comment by SurvivCreator** [comment]
kay what sound effect do you prefer? Samsung Whistle or Apple Tritone? (for the notification).
Also, do you think we can add mobile? Seems like this is the most viewed game :D

**Comment by Coolaction77** [comment]
Personally I'd say Apple tri tone only higher pitched.

**Comment by SurvivCreator** [comment]
kk I will work on that for 500 view special :D

**Comment by Coolaction77** [comment]
Cool!

**Comment by Coolaction77** [comment]
I think we should move the achievement stuff around, it looks kinda weird.
